### PR TITLE
patroni to pgo migration

### PR DESCRIPTION
**Pull request by pbastia**

### CODE REVIEW

File: helm/cas-metabase/templates/cron-prod-test-restore.yaml [hunk 1]
```diff
@@ -26,23 +26,37 @@ spec:
           containers:
             - env:
                 - name: MB_USER
-                  value: metabase
+                  alueFrom:
```
Comment (dleard) [inline helm/cas-metabase/templates/cron-prod-test-restore.yaml:29]:
```suggestion
                  valueFrom:
```

File: helm/cas-metabase/values.yaml [hunk 2]
```diff
@@ -86,7 +87,26 @@ download-cas-metabase-dags:
...
+
+  deployment:
+    name: cas-metabase
+    # for the KNP
+    sourceReleaseName: cas-metabase
+    targetReleaseName: cas-metabase-db
+
+  from:
+    secretName: cas-metabase-patroni
+    host: cas-metabase-patroni.9212c9-dev.svc.cluster.local
```
Comment (dleard) [inline helm/cas-metabase/values.yaml:104]: Should "host" be set to ~ & specified in each values-environment yaml file?

**Review by dleard — CHANGES_REQUESTED** [review]
Looks great! A couple of small changes needed

**Review by dleard — APPROVED** [review]
:100: Let me know when you want to run this!

**Comment by pbastia** [comment]
@dleard it's been run on -dev. It should automatically run on -test upon merging to `main`, we'll need to monitor that closely

**Pull request merged**